devops/setup-gpg/action.yaml

```diff
@@ -1,52 +1,72 @@
 name: 'Setup GPG'
 description: 'Configures this action to run gpg with a given key and pass'
 inputs:
-  gpg-private-key:  # id of input
+  gpg-private-key:
+    description: 'GPG private key exported as an ASCII armored version or its base64 encoding'
     required: true
-  gpg-key-pass:  # id of input
+  gpg-key-pass:
+    description: 'Passphrase of the GPG private key'
     required: true
-  gpg-key-name:  # id of input
+  gpg-public-key:
+    description: 'GPG public key exported as an ASCII armored version or its base64 encoding'
     required: true
-    default: "Aerospike"
+  gpg-key-name:
+    description: 'GPG key name'
+    required: true
+    default: "aerospike-inc"
+  gpg-trust-level:
+    description: 'Set key trust level'
+    required: false
+    default: 5
 runs:
   using: "composite"
   steps:
-      - name: "check if private key is not empty"
+      - name: "check if private key is empty"
         env: 
-            PRIVATE_KEY: ${{ inputs.gpg-private-key }}
-        if: ${{ env.PRIVATE_KEY == '' }}
+            GPG_PRIVATE_KEY: ${{ inputs.gpg-private-key }}
+        if: ${{ env.GPG_PRIVATE_KEY == '' }}
+        run: |
+          echo "The gpg-private-key was empty"
+          exit 1
+        shell: bash
+      - name: "check if public key is empty"
+        env:
+            GPG_PUBLIC_KEY: ${{ inputs.gpg-public-key }}
+        if: ${{ env.GPG_PUBLIC_KEY == '' }}
         run: |
-          echo "the gpg-private-key was empty"
+          echo "The gpg-public-key was empty"
           exit 1
         shell: bash
-      - name: "check if key name is not empty"
+      - name: "check if key name is empty"
         env: 
-            KEY_NAME: ${{ inputs.gpg-key-name }}
-        if: ${{ env.KEY_NAME == '' }}
+            GPG_ID: ${{ inputs.gpg-key-name }}
+        if: ${{ env.GPG_ID == '' }}
         run: |
-          echo "the gpg-key-name was empty"
+          echo "The gpg-key-name was empty"
           exit 1
         shell: bash
-      - name: "check if key pass is not empty"
+      - name: "check if key pass is empty"
         env: 
-            KEY_PASS: ${{ inputs.gpg-key-pass }}
-        if: ${{ env.KEY_PASS == '' }}
+            GPG_KEY_PASS: ${{ inputs.gpg-key-pass }}
```

Comment:
renaming should be it's own PR. this PR is all over the place.

Reply:
right. you made a lot of changes that where not "make it work" 

the only thing that I see that "makes it work" is file permission changes to verify... Adding dpkg-sig was not even something to "make it work" merely a nice to have. So when I look back at the commit message will it be clear what and why something was changed? I can take it from here though! no problem! Thank you!

Reply:
This example was for setup-gpg.
What is now working here allows for *.rpm, as well as *.deb files to be signed.
I think this is exactly what our devs would want to have.
Not sure why it would need to be split up into smaller parts.
Although, I will leave that to you to sort out.